docs/contributing/contribution/development.md

```diff
@@ -0,0 +1,188 @@
+- [ ] This change requires a documentation update
+
+# How Has This Been Tested?
+
+Please describe the tests that you ran to verify your changes. If applicable, provide instructions so others can reproduce the results. Please also list any relevant details for your test configuration
+
+- [ ] Test A
+- [ ] Test B
+
+For bugs, if available, provide the following information:
+- **Reporter Name**: 
+- **Reported Date**:  
+- **Steps to reproduce**: As specific as possible
+- **Release/Build Version**: 
+- **Environment**: OS, framework version, etc
+
+# Checklist:
+
+- [ ] My code follows the style guidelines of this project
+- [ ] I have performed a self-review of my own code
+- [ ] I have commented on my code, particularly in hard-to-understand areas
+- [ ] I have made corresponding changes to the documentation
+- [ ] My changes generate no new warnings
+- [ ] I have added tests that prove my fix is effective or that my feature works
+- [ ] New and existing unit tests pass locally with my changes
+- [ ] Any dependent changes have been merged and published in downstream modules
+```
+
+Make sure to complete all required fields with the following information:
+- **Reviewers**: Provide at least 2 reviewers to ensure you get enough approvals
+- **Milestone**: Link the PR to the corresponding Milestone
+- **Linked Issues**: Link the PR to the corresponding Issue
+
+## Testing Phase
+
+After a PR has been created, wait for the automation test suite to complete. it is desirable to have some community members review and test locally the PR, by providing feedback on the PR comments.
+
+## Review Phase
+
+Wait a couple of days to collect comments from other contributors. Make sure to respond, eventually fix, and mark all the comments as done.
+Wait for the commenter to acknowledge the answer or the fix of its comment.
+
+## Actors Approvals
+
+Ping the required actors on a PR comment to call their attention after this step has been fulfilled to ask for their final review. Take into consideration that these actors are busy and will ignore any PRs that do not have the previous checks. Follow the actor's comments and provide feedback for the fix. Make sure to mention them on any change related to their comments.
+
+## Merge Blockers
+
+- Issue PRs will be selected for running a subset of the Testing Regression suite and require all test to be green before being merged.
+- Issue PRs will require having all the comments reviewed and checked by both commenter and developer.
+- Issues PRs will require at lease 2 approvals from any contributor to unblock merging to the integration branch.
+
+## Merge and Issue Close
+
+Once the PR has been reviewed, and all the PR blockers are green, go ahead and merge the PR to the integration branch. After the PR is merged, make sure that the issue is marked as closed and some comments related to the final result are included.
```

Comment:
Any ideas on this? I think it will slow down the process. We can automate and send a notification on the goal chat when a PR is closed, will that work?

Reply:
I suppose it will, at least for the time being, will add to the backlong of ideas to discuss